CheatSheets/windows-cheatsheet.md

```diff
@@ -46,6 +64,15 @@
 | `Win + E` | Open File Explorer |
 | `Win + ,` | see your desktop until you hold keys |
 | `Win + v` | to see clipboard and emoji |
+| `Win + .`| Open Emojis section |
```

Comment:
This command already exists in the `App Rearrangement Shortcuts (Metro)` section.